dev-tools/kubernetes/filebeat/manifest.run.yaml

```diff
@@ -9,9 +112,16 @@ metadata:
 data:
   filebeat.yml: |-
     filebeat.inputs:
-    - type: container
+    - type: filestream
```

Comment:
thanks for the suggestion. Does it means this line change should be applied anywhere we use the filebeat.input including
- deploy/kubernetes/filebeat-kubernetes.yaml
- deploy/kubernetes/filebeat/filebeat-configmap.yaml

since I copied the config under dev-tools from those files

Reply:
I am addressing your concerned in the linked PR